content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md

```diff
@@ -0,0 +1,651 @@
+---
+api_metadata:
+  apiVersion: "rbac.authorization.k8s.io/v1"
+  import: "k8s.io/api/rbac/v1"
+  kind: "ClusterRoleBinding"
+content_type: "api_reference"
+description: "ClusterRoleBinding 引用 ClusterRole，但不包含它。"
+title: "ClusterRoleBinding"
+weight: 6
+auto_generated: true
+---
+
+<!-- 
```

Comment:
There are many trailing spaces in this file, please check.

Reply:
hmmm, removed much. zh followed the en source and has many trailing spaces too -_!